proto/minder/v1/minder.proto

```diff
@@ -2756,3 +2758,11 @@ message UpstreamEntityRef {
     // be stored in Minder.
     google.protobuf.Struct properties = 3;
 }
+
+// BundleSubscription represents a subscription to a bundle.
+message BundleSubscription {
+    string id = 1;
+    string project_id = 2;
+    string bundle_id = 3;
+    string current_version = 4;
+}
```

Comment:
It feels like this should be someplace different from the external API, since it's not referenced by any clients.

We have an `internal/proto`, but maybe it makes sense to have a pkg/proto (and exclude that from coverage)?